src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/activities/ActivitiesMigrationResource.kt

```diff
@@ -124,4 +126,107 @@ class ActivitiesMigrationResource(
       includeOnlyFailures = includeOnlyFailures,
     ),
   )
+
+  @PreAuthorize("hasRole('ROLE_MIGRATE_ACTIVITIES')")
+  @GetMapping("/activities/history/{migrationId}")
+  @Operation(
+    summary = "Gets a specific migration history record",
+    description = "Requires role <b>MIGRATE_ACTIVITIES</b>",
+    responses = [
+      ApiResponse(
+        responseCode = "200",
+        description = "The migration history record",
+        content = [
+          Content(
+            mediaType = "application/json",
+            schema = Schema(implementation = MigrationHistory::class),
+          ),
+        ],
+      ),
+      ApiResponse(
+        responseCode = "401",
+        description = "Unauthorized to access this endpoint",
+        content = [Content(mediaType = "application/json", schema = Schema(implementation = ErrorResponse::class))],
+      ),
+      ApiResponse(
+        responseCode = "403",
+        description = "Incorrect permissions to access this endpoint",
+        content = [Content(mediaType = "application/json", schema = Schema(implementation = ErrorResponse::class))],
+      ),
+      ApiResponse(
+        responseCode = "404",
+        description = "Migration not found",
+        content = [Content(mediaType = "application/json", schema = Schema(implementation = ErrorResponse::class))],
+      ),
+    ],
+  )
+  suspend fun get(
+    @PathVariable
+    @Schema(description = "Migration Id", example = "2020-03-24T12:00:00", required = true)
```

Comment:
`required` is deprecated - should be using `RequiredMode` these days.  Didn't Andy also suggest that this field got filled in based off the kotlin type recently too?